__mocks__/@glif/react-components.js

```diff
@@ -33,3 +39,57 @@ async function getAddress(variables) {
 }
 
 export const useAddressLazyQuery = () => [getAddress]
+
+export const WalletProviderContext = createContext({
+  ...walletProviderInitialState
+})
+
+const WalletProviderWrapper = ({
+  children,
+  options,
+  initialState,
+  getState
+}) => {
+  const [walletProviderState, walletProviderDispatch] = useReducer(
+    options?.reducer || walletProviderReducer,
+    initialState
+  )
+
+  const mockWalletProviderContextFuncs = createMockWalletProviderContextFuncs(
+    options?.walletProviderDispatch || walletProviderDispatch,
+    walletProviderState
+  )
+
+  return (
+    <WalletProviderContext.Provider
+      value={{
+        // @ts-ignore
+        state: walletProviderState,
+        dispatch: options?.walletProviderDispatch || walletProviderDispatch,
+        ...mockWalletProviderContextFuncs
+      }}
+    >
+      <>
+        {getState(walletProviderState)}
+        {children}
+      </>
+    </WalletProviderContext.Provider>
+  )
+}
+
+export const useWalletProvider = () => {
+  const value = useContext(WalletProviderContext)
+  const { state } = value
+  return {
+    ...state,
+    ...value
+  }
+}
+
+export function useWallet() {
+  const { wallets, selectedWalletIdx } = useWalletProvider()
+
+  if (wallets.length === 0) return noWallet
+  if (!wallets[selectedWalletIdx]) return noWallet
```

Comment:
@Schwartz10 `noWallet` here is not defined, it's the same for the `react-components` mock in the `wallet` repo. Currently it's hitting one of these clauses due to another problem with the provider mocking I assume, but I don't think there should be any undefined values here, as that is what it's complaining about now

Reply:
Oof sorry about that, not sure how the import didn't make it over